rust/BUILD

```diff
@@ -90,6 +92,17 @@ deploy_crate(
     target = ":assemble_crate",
 )
 
+# Documentation generation
+rust_docs_parser(
+    name = "driver_adoc_async",
+    feature = "async",
+)
+
+rust_docs_parser(
+    name = "driver_adoc_sync",
+    feature = "sync",
+)
+
```

Comment:
We generate two versions of documentation: `async` and `sync` in order to have both variants of some methods.